DataLoader.py

```diff
@@ -16,7 +16,7 @@ def __init__(self, file_path):
                     self.abstract.append(entry['text'])
                 # Body text
             else:
-                content['abstract'] = None
+                content['abstract'] = 'No abstract provided'
```

Comment:
Go ahead and make the first N lowercase that way the regex used later doesn't need to replace it.